pkg/server/api/enroll.go

```diff
@@ -151,6 +151,14 @@ func (c *Api) sign(keyId string, principals []string, serial uint64, certType ui
 		Permissions:     getPermissionsForCertType(&c.conf.SSH, certType),
 	}
 
+	if c.conf.Github.Enabled {
+		gitUser, err := c.RetrieveGitUsername(keyId)
```

Comment:
Mapping is optional in a sense that some users will have a github username and other won't - and this is perfectly fine (it's not even an edge case) - but seems that the code here requires, and would fail, if mapping does not exist.

Reply:
not sure what you mean? logger.Error should only log an error and continue with an empty github username